### PR TITLE
Roc 8068 pcq terraform

### PR DESCRIPTION
**Pull request by yogendraU**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-8068

### Change description
This JIRA is extension of ROC-8085

- Add PCQ Integration flag in cmc-citizen-front-end code base
- Add secret for getting pcq token in cmc-citizen-front-end code base
- Add pcq feature as environment variable

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: charts/cmc-citizen-frontend/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
-apiVersion: v1
+apiVersion: v2
```
Comment (klawani) [inline charts/cmc-citizen-frontend/Chart.yaml:1]: try to rebase with master now ... otherwise you will get conflict

Comment (yogendraU) [reply charts/cmc-citizen-frontend/Chart.yaml:1]: rebase with master

**Comment by andrewwa-kainos** [comment]
Please update the PR with the details. JIRA link, a Change Description, etc

**Comment by yogendraU** [comment]
> Please update the PR with the details. JIRA link, a Change Description, etc

Thanks Andy. Have updated the PR with Jira Link and Description.

**Comment by yogendraU** [comment]
> LGTM, however didn't see it's usage of new feature, may be in another PR

Usage of this feature is part of pcq-integration front end code changes done by Prathap.
https://github.com/hmcts/cmc-citizen-frontend/compare/Features/ROC-8086-PCQ-Integration

**Pull request merged**